x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source.scss

```diff
@@ -98,3 +98,9 @@
     background: transparent;
   }
 }
+
+@media (min-width: 768px) {
+  .organizational-content-source-item {
+    max-width: calc(33.3% - 16px);
```

Comment:
You'll wanna replace the `16px` here with an EUI size variable — in this case it'll be the default `$euiSize`

Reply:
Thanks. Yes I did try to use `$euiSize` in place of `16px` but the variable doesn't seem to be available when built and viewed in the browser. It shows as missing and does not render:
<img width="290" alt="Screenshot 2022-01-26 at 11 42 49" src="https://user-images.githubusercontent.com/7115017/151157573-6f3a3f31-6473-4997-bdd2-c71468040377.png">

Reply:
Ah, resolved, need to wrap the variable with `#{...}`.